packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -28,155 +36,95 @@ export const Default = {
 };
 
 /**
- * The marker in its 'Cluster' variant in all possible states.
+ * Use the `variant` attribute to change the appearance of the marker.
+ *
+ * - `main`: The default variant.
+ * - `place`: A variant that is used to show a location on the map.
+ * - `cluster`: A variant that is used to show multiple locations on the map.
  */
-export const VariantCluster = {
-  name: 'Variant Cluster',
-  parameters: { controls: { exclude: ['state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state', values: ['default', 'hover'] }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: '8'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'cluster'
-        }
-      ]
-    });
-  }
+export const Variant = {
+  render: () => html`
+    <div class="flex items-center gap-12">
+      <sd-map-marker></sd-map-marker>
+      <sd-map-marker variant="place">
+        <sd-icon name="content/image" color="primary"></sd-icon>
+      </sd-map-marker>
+      <sd-map-marker variant="cluster"> 8 </sd-map-marker>
+    </div>
+  `
 };
 
 /**
- * The marker in its 'Main' variant in all possible states.
+ * Use the `state` attribute to change the appearance of the marker.
+ *
+ * - `default`: The default state.
+ * - `hover`: The state when the marker is hovered.
+ * - `active`: The state when the marker is active.
  */
-export const VariantMain = {
-  name: 'Variant Main',
-  parameters: { controls: { exclude: ['default', 'state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state' }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: ''
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'main'
-        }
-      ]
-    });
-  }
-};
-
-/**
- * The marker in its 'Place' variant in all possible states.
- */
-export const VariantPlace = {
-  name: 'Variant Place',
-  parameters: { controls: { exclude: ['default', 'state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state' }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: '<sd-icon name="content/image" color="primary"></sd-icon>'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'place'
-        }
-      ]
-    });
-  }
-};
+export const State = {
+  render: () => html`
```

Comment:
![image](https://github.com/user-attachments/assets/623bb8f3-e53b-4b4f-a7ff-b40afd38eb99)

The cluster cannot be active I guess ...

Reply:
Updated